### PR TITLE
add example files with settings which are not covered, generated with cavif

### PR DESCRIPTION
**Pull request by ledyba-z**
Hi! We are [Link-U](https://www.link-u.co.jp/), a Japanese digital comics publishing company.

We are strongly interested to use AVIF in production.

I found that there are no example images in some settings, and generated:

 - [YUV422 images](https://github.com/link-u/avif-sample-images#yuv422)
 - [10bit/12bit monochrome images](https://github.com/link-u/avif-sample-images#avif-version)
 - [Images with mirroring transformation](https://github.com/link-u/avif-sample-images#mirroring-horizontally)
 - [Images with cropping (`clap` property)](https://github.com/link-u/avif-sample-images#cropping)
 - [Images with multiple transformation](https://github.com/link-u/avif-sample-images#mirroring-vertically--rotating-at-90-degrees)
    - [This image](https://github.com/link-u/avif-sample-images#cropping--mirroring-vertically--rotating-at-90-degrees) requires to apply cropping, rotation, mirroring to display correctly in this order, as [MIAF](https://www.iso.org/standard/74417.html) says.

We are developing [encoder](https://github.com/link-u/cavif) and [decoder](https://github.com/link-u/davif) commands ourself, and generated those examples by them.

I checked that all images can be decoded by libavif (but libavif does not support transformation yet).

Please take a look!

### CODE REVIEW

**Comment by ledyba-z** [comment]
I added images with odd-dimensions.

They cannot be opened in Windows AV1 Video Extension, but can be decoded by libavif or [davif](https://github.com/link-u/davif). If I understand you correctly, [AV1 spec](https://aomediacodec.github.io/av1-spec/av1-spec.pdf) allows both odd-width and odd-height.

**Comment by cconcolato** [comment]
Thank you for the PR. Sorry for the delay in reviewing it. The group still needs some time to review your files. We will let you know asap.

**Comment by ledyba-z** [comment]
Thanks a lot!

`hato.profile2.12bpc.yuv422.monochrome.avif` should be monochrome, but isn't.
So I replace (and push) it.

File: testFiles/Link-U/README.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+## Kimono - Transformation tests
+
+[<img src="https://raw.githubusercontent.com/link-u/avif-sample-images/master/kimono.jpg" alt="kimono.jpg" height="512">](kimono.jpg)
+
+ - size: 722x1024
+ - By Momiji Jinzamomi([@momiji-san](https://github.com/momiji-san)) and Kaede Fujisaki ([@ledyba](https://github.com/ledyba))
+ - Retrieved from [their website](https://hexe.net/2018/12/24/18:59:01/).
+
+Test images for rotation(`irot`), mirroring(`imir`), cropping(`clap`).
```
Comment (cconcolato) [inline testFiles/Link-U/README.md]: According to MIAF, "All transformative properties associated with coded and derived images required or conditionally required by this document shall be marked as essential" but that's not the case in your files. Note that I used https://gpac.github.io/ComplianceWarden-wasm/ to validate some of the files and to find this issue.

File: testFiles/Link-U/README.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
+# AVIF Example files from [Link-U](https://www.link-u.co.jp/)
+
+ - All files do not contain Exif metadata.
+ - All files are tagged as MIAF compatible.
+ - All files are tagged as compatible with the AVIF Baseline or Advanced Profile if possible.
+ - All images have the "reduced_still_picture_header" and "still_picture" flags set to 1 in the AV1 Sequence Header.
+ - All images are licensed under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en).
```
Comment (cconcolato) [inline testFiles/Link-U/README.md]: I understand that you produced the AVIF files and provide them with a CC-BY-SA license, but I see that you retrieved the images from some web site. Can you confirm also that the source images have the same license?

File: testFiles/Link-U/README.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+| 1       | 8         | YUV444  |            | YES       |            | [here](fox.profile1.8bpc.yuv444.odd-width.avif)                        |
+| 1       | 8         | YUV444  |            |           | YES        | [here](fox.profile1.8bpc.yuv444.odd-height.avif)                       |
+| 1       | 8         | YUV444  |            | YES       | YES        | [here](fox.profile1.8bpc.yuv444.odd-width.odd-height.avif)             |
+| 1       | 10        | YUV444  |            |           |            | [here](fox.profile1.10bpc.yuv444.avif)                                 |
+| 1       | 10        | YUV444  |            | YES       |            | [here](fox.profile1.10bpc.yuv444.odd-width.avif)                       |
+| 1       | 10        | YUV444  |            |           | YES        | [here](fox.profile1.10bpc.yuv444.odd-height.avif)                      |
+| 1       | 10        | YUV444  |            | YES       | YES        | [here](fox.profile1.10bpc.yuv444.odd-width.odd-height.avif)            |
+| 1       | 12        | YUV444  |            |           |            | [here](fox.profile2.12bpc.yuv444.avif)                                 |
```
Comment (cconcolato) [inline testFiles/Link-U/README.md]: From this point, the profile should be 2 because of 12bpc.

File: testFiles/Link-U/README.md (hunk 2 above)
Comment (ledyba) [reply testFiles/Link-U/README.md]: Hello, I am an author of original images. I added the license badges to pages. Please check.

https://hexe.net/2017/11/27/12:27:02/
https://hexe.net/2018/12/24/18:59:01/
https://hexe.net/2017/12/02/16:33:53/

File: testFiles/Link-U/README.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+| 0       | 8         |            | NO   | YES              | [here](hato.profile0.8bpc.yuv420.no-cdef.avif)                            |
+| 0       | 8         | YES        | NO   | YES              | [here](hato.profile0.8bpc.yuv420.monochrome.no-cdef.avif)                 |
+| 0       | 10        |            | NO   | NO               | [here](hato.profile0.10bpc.yuv420.no-cdef.no-restoration.avif)            |
+| 0       | 10        | YES        | NO   | NO               | [here](hato.profile0.10bpc.yuv420.monochrome.no-cdef.no-restoration.avif) |
+
+#### YUV422
+
+| profile | bit depth | Monochrome | CDEF | Loop Restroation | file                                                                      |
+|---------|-----------|------------|------|------------------|---------------------------------------------------------------------------|
+| 0       | 8         |            | NO   | YES              | [here](hato.profile2.8bpc.yuv422.no-cdef.avif)                            |
```
[the remark this answers is not in the document]
Comment (ledyba-z) [reply testFiles/Link-U/README.md]: You are right! I fixed the tables.

File: testFiles/Link-U/README.md (hunk 3 above)
Comment (ledyba-z) [reply testFiles/Link-U/README.md]: Thanks, I also fixed this table.

File: testFiles/Link-U/README.md (hunk 2 above)
Comment (ledyba-z) [reply testFiles/Link-U/README.md]: I contacted the authors and asked them to specify license.

File: testFiles/Link-U/README.md (hunk 1 above)
Comment (ledyba-z) [reply testFiles/Link-U/README.md]: Thanks! I re-encode all `kimono` images and mark `irot`, `imir` and `clap` as essential.
(And also, I validated those images using https://gpac.github.io/ComplianceWarden-wasm/ )

**Comment by ledyba-z** [comment]
@cconcolato 
Thank you for your review!

I updated README.md and transformation test images(`kimono`).
And also, I contacted the image author (my sister) and asked her to make the license clear.

Please take another look.

**Pull request merged**